src/Polkadot/Extrinsic.h

```diff
@@ -41,8 +43,10 @@ class Extrinsic {
         if (input.has_era()) {
             era = encodeEra(input.era().block_number(), input.era().period());
         } else {
-            era = encodeCompact(0);
+          // immoral era
```

Comment:
'immortal' instead of 'immoral'. :D , or just 'era'

Reply:
😂 will fix this typo